### DISCUSSION TITLE
POST request with formFile data, Failed to get the content.

### DISCUSSION THREAD

**Discussion by jimMatterless** [pocketbase/pocketbase#4387] [Q&A] [answered]
I created a post request, that uploads a file by formData.
In Pocketbase, I can read the file header, getting the filename and size.
But is there any way to read the content of the file?
ps: the file I upload is just a txt

```
const [mpfile, mpheader] = c.request().formFile("data");
const file = $filesystem.fileFromMultipart(mpheader);
console.log(JSON.stringify(file));
```
I don't want to directly upload the file but modify the data and add it as a record. But I couldn't get the content.
Please help.

**Comment by ganigeorgiev** [comment, marked as answer]
You can try with the [`readerToString()`](https://pocketbase.io/jsvm/functions/readerToString.html) JSVM helper. Something like:

```js
const [mpfile, mpheader] = c.request().formFile("data");
try {
    const content = readerToString(mpfile)
    console.log(content)
} finally {
    mpfile.close()
}
```

**Reply by jimMatterless** [reply]
It works! Thank you very much!!
